src/app/zap-templates/zcl/data-model/chip/operational-credentials-cluster.xml

```diff
@@ -32,6 +32,8 @@ limitations under the License.
     <description>This cluster is used to add or remove Operational Credentials on a Commissionee or Node, as well as manage the associated Fabrics.</description>
 
     <attribute side="server" code="0x0001" define="FABRICS" type="ARRAY" entryType="FabricDescriptor" length="254" writable="false" optional="false">fabrics list</attribute>
+    <attribute side="server" code="0x0002" define="SUPPORTED_FABRICS" type="INT8U" writable="false" optional="false">supported fabrics</attribute>
+    <attribute side="server" code="0x0003" define="COMMISSIONED_FABRICS" type="INT8U" writable="false" optional="false">commissioned fabrics</attribute>
```

Comment:
In the spec the name is `CommissionedFabrics ` should we use that instead of `commissioned fabrics` ?